src/genie_python/genie.py

```diff
@@ -2554,3 +2554,15 @@ def get_detector_table() -> str | None:
     """
     assert _genie_api.dae is not None
     return _genie_api.dae.get_table_path("Detector")
+
+
+@usercommand
+@log_command_and_handle_exception
+def change_autosave(freq: float) -> None:
+    """Change the rate of ICP autosave
+
+    Args:
+        freq (float): frequency of autosave in Hz.
+    """
```

Comment:
The units are actually "Frames" (as in isis pulses) as opposed to Hz and are an integer